cas-server-documentation/installation/LDAP-Authentication.md

```diff
@@ -10,12 +10,12 @@ LDAP integration is enabled by including the following dependency in the Maven W
 <dependency>
      <groupId>org.jasig.cas</groupId>
      <artifactId>cas-server-support-ldap</artifactId>
-     <version>${cas.version}</version>
+     <version>${project.version}</version>
```

Comment:
This should be reverted. Project version implies that it should the version of the maven overlay. We want the cas version here stated explicitly. and it's something that all other docs use as well. 

Otherwise, gorgeous. Thank you.